app/main.py

```diff
@@ -89,7 +85,6 @@ def end():
     return end_response()
 
 
-
 def boardToArray(dataDump):
```

Comment:
So I'm not sure how you've done it but this function has already been moved to parsableBoard and now is back in main.py 

Easiest solution I see is to copy paste this one overtop of the the parsableBoard representation of it (if you've made changes) or just simply remove the code from main.py